ros-ocp/templates/authorino-services.yaml

```diff
@@ -0,0 +1,70 @@
+{{- if eq (include "ros-ocp.jwt.shouldEnable" .) "true" }}
+---
+# Main gRPC authorization service used by Envoy ext_authz
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "ros-ocp.fullname" . }}-authorino-authorization
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "ros-ocp.labels" . | nindent 4 }}
+    app.kubernetes.io/component: authorino
+    app.kubernetes.io/part-of: ros-ocp
+spec:
+  selector:
+    {{- include "ros-ocp.selectorLabels" . | nindent 4 }}
+    app.kubernetes.io/component: authorino
+  ports:
+  - name: grpc
+    port: 50051
+    targetPort: 50051
+    protocol: TCP
```

Comment:
both `authorino-services.yaml` and `authorino-tls-cert.yaml` expose port `50051` to the same authorino pods (they have the same selector `app.kubernetes.io/component: authorino`). 
Is this intentional? Why?